models/articles.model.js

```diff
@@ -0,0 +1,23 @@
+const db = require("../db/connection");
+
+function selectArticleId(id) {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1", [id])
+    .then((result) => {
+      const articles = result.rows;
+      if (articles.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: `No article found for article id: ${id}`,
+        });
+      }
+      return articles;
```

Comment:
For this one do you want to return an array containing a single article or just the 1 article object the user is looking for?